### PR TITLE
UPSTREAM: 50620: allow heapster option defaults to be set

### PR DESCRIPTION
**Pull request by juanvallejo**
Related BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1470003
Patches upstream `node_top.go` and `pod_top.go` to receive custom heapster option value defaults.

**Before**
```
$ oc adm top node --heapster-namespace=openshift-infra --heapster-scheme=https
...
```

**After**
```
# flags to set openshift-specific values no longer needed if using default configuration
$ oc adm top node
...
```

cc @openshift/cli-review

### CODE REVIEW

File: pkg/oc/admin/top/top.go [hunk 1]
```diff
@@ -12,7 +12,14 @@ import (
...
 )
 
-const TopRecommendedName = "top"
+const (
+	TopRecommendedName = "top"
+
+	DefaultHeapsterNamespace = "openshift-infra"
+	DefaultHeapsterScheme    = "https"
+	DefaultHeapsterService   = "heapster"
+	DefaultHeapsterPort      = "" // use the first exposed port on the service
```
Comment (fabianofranz) [inline pkg/oc/admin/top/top.go]: Doesn't seem to be used anywhere.

**Comment by fabianofranz** [comment]
A couple tests if possible.  ;)

**Comment by juanvallejo** [comment]
@fabianofranz 
> A couple tests if possible. ;)

Thanks, done! Will add same tests to upstream PR

**Comment by juanvallejo** [comment]
Upstream PR has merged: https://github.com/kubernetes/kubernetes/pull/50620

**Comment by juanvallejo** [comment]
cc @smarterclayton

**Pull request merged**